doc/classes/NavigationObstacle2D.xml

```diff
@@ -1,42 +1,71 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<class name="NavigationObstacle2D" inherits="Node" is_experimental="true" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+<class name="NavigationObstacle2D" inherits="Node2D" is_experimental="true" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		2D Obstacle used in navigation for collision avoidance.
+		2D Obstacle used in navigation to constrain avoidance controlled agents outside or inside an area.
```

Comment:
This wording is difficult to parse. May I suggest "A dynamic 2D obstacle used to restrict the pathfinding of navigation agents." and likewise for 3D?

Reply:
It does not affect the pathfinding. Pathfinding is its own distinct thing.
What the obstacle does is it constrains the possible avoidance velocities to an area.